layouts/partials/posts_navigation.html

```diff
@@ -1,15 +1,23 @@
 {{ $skipPageNav := .Scratch.Get "skipPageNav" }}
 {{ if not (or .Params.skip_pagenav $skipPageNav) }}
-    {{ if or .NextInSection .PrevInSection }}
+    {{$prev := .PrevInSection}}
+    {{$next := .NextInSection}}
+    {{ with .Params.prevInSection }}
+        {{$prev = .}}
+    {{ end }}
+    {{ with .Params.nextInSection }}
+        {{$next = .}}
+    {{ end }}
+    {{ if or $prev $next }}
```

Comment:
We've always had the pagenav display even if there's nothing in it.
More worrying is you didn't seem to close the `{{ if }}` block.